src/chainstate/stacks/boot/lockup.clar

```diff
@@ -1,145 +1,6 @@
-;; The .lockup boot contract
-;; Error codes
-(define-constant ERR_UNLOCK_UNREACHABLE 255)
+(define-map vesting-schedules uint (list 4426 { recipient: principal, amount: uint }))
```

Comment:
We need to avoid using the word "vesting" when describing these locked tokens, since it has legal implications.  The tokens are not being vested -- they are already owned (and owed by us) to their respective address owners.  This code is meant to unlock them.

It was unfortunate that we called it "vesting" in Stacks 1.0, since this was inaccurate.

Reply:
This applies to Clarity and to the Rust code.

Reply:
Will fix!

Reply:
addressed with https://github.com/blockstack/stacks-blockchain/pull/2109/commits/52fc632b8e594b00a579daf9c6a7e2f31c76f6bc (thank you @zone117x)